### PR TITLE
build: add DragonFly BSD support in autotools

### PR DESCRIPTION
**Pull request by rolinh**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -271,6 +271,11 @@ libuv_la_SOURCES += src/unix/darwin.c \
                     src/unix/proctitle.c
 endif
 
+if DRAGONFLY
+include_HEADERS += include/uv-bsd.h
+libuv_la_SOURCES += src/unix/kqueue.c src/unix/freebsd.c
```
Comment (saghul) [inline Makefile.am:276]: Mind reversing thr order of the files? Just like the freebsd section below.

Comment (rolinh) [inline Makefile.am:276]: I can, of course, but note that only FreeBSD has it the other way around (OpenBSD and NetBSD are ordered this way too).

Comment (saghul) [inline Makefile.am:276]: Ah, leave it then :-)

**Comment by saghul** [comment]
Thanks Robin! Landed in 3193b67

**Pull request closed without merging**